BookPlayer/Library/ItemListViewController.swift

```diff
@@ -138,9 +143,24 @@ class ItemListViewController: UIViewController, ItemList, ItemListAlerts, ItemLi
         self.animateView(self.bulkControls, show: editing)
         self.tableView.setEditing(editing, animated: true)
 
+        if editing {
+            previousLeftButtons = navigationItem.leftBarButtonItems
+            navigationItem.leftBarButtonItems = [selectButton]
+            selectButton.isEnabled = tableView.numberOfRows(inSection: 0) > 0
+            updateSelectButtonTitle()
+        } else {
+            navigationItem.leftBarButtonItems = previousLeftButtons
+        }
+
         NotificationCenter.default.post(name: notification, object: nil, userInfo: nil)
     }
 
+    func updateSelectButtonTitle() {
+        self.selectButton.title = self.tableView.numberOfRows(inSection: 0) > (self.tableView.indexPathsForSelectedRows?.count ?? 0)
```

Comment:
also let's add a `guard` here at the beginning of the function to avoid doing this when it's not editing
```
guard self.tableView.isEditing else { return }
```